File: src/main/java/com/p2p/lending/service/PoundageService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Poundage;


public interface PoundageService {
	Map<String, Object> selectpa(String currpge, Map<String, Object> findmap);
	List<Poundage> findList(Map<String, Object> map);
	int insert(Poundage po);
}

File: src/main/java/com/p2p/lending/service/PoundageService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Poundage


interface PoundageService {
    fun selectpa(currpge: String, findmap: Map<String, Any>): Map<String, Any>
    fun findList(map: Map<String, Any>): List<Poundage>
    fun insert(po: Poundage): Int
}
